packages/lib-classifier/src/store/Classification/Classification.js

```diff
@@ -19,6 +20,17 @@ const Classification = types
     metadata: types.maybe(ClassificationMetadata)
   })
   .views(self => ({
+    /**
+     * Returns false until we start updating task annotations.
+     */
+    get inProgress() {
+      let inProgress = false
+      self.annotations.forEach(annotation => {
+        inProgress ||= annotation._inProgress || annotation._choiceInProgress
```

Comment:
`_choiceInProgress` is used by the survey task. All other tasks use `_inProgress`.